src/connectionsetup.ts

```diff
@@ -0,0 +1,153 @@
+/* --------------------------------------------------------------------------------------------
+ * SonarLint for VisualStudio Code
+ * Copyright (C) 2017-2022 SonarSource SA
+ * sonarlint@sonarsource.com
+ * Licensed under the LGPLv3 License. See LICENSE.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+'use strict';
+
+import * as vscode from 'vscode';
+import { ConfigurationTarget } from 'vscode';
+import { Commands } from './commands';
+import { ResourceResolver } from './webview';
+import * as util from './util';
+
+let connectionSetupPanel: vscode.WebviewPanel;
+
+export function connectToSonarQube(context: vscode.ExtensionContext) {
+  return () => {
+    lazyCreateConnectionSetupPanel(context);
+    connectionSetupPanel.webview.html = renderConnectionSetupPanel(context, connectionSetupPanel.webview);
+    connectionSetupPanel.webview.onDidReceiveMessage(handleMessage);
+    connectionSetupPanel.iconPath = {
```

Comment:
From WebViewPanel - iconPath?: Uri | { light: Uri; dark: Uri };
so you can only specify `util.resolveExtensionFile('images/sonarqube.svg')` since values for both `light` and `dark` are the same